Update README.md (#50)
lachenlama/ez_Unix: README.md

Celebrate [Hacktoberfest](https://hacktoberfest.digitalocean.com/) by getting involved in the open source community by completing some simple tasks in this project. 

## Repository's Aim

> * The repository is for Unix commands and their uses. The contributors may add what they wish within the context of being used in Unix CLI. 

Reviewer: ```suggestion
* The repository is for Unix commands and their uses. Contributors can add whatever they wish within the context of using Unix CLI. 
```
Slight rephrasing for clarity and conciseness.

> * We recommend the contributors to use CLI to clone, write code and push your code to this repository.

Reviewer: ```suggestion
* We recommend the contributors to use CLI to clone, write code, and push your code to this repository.
```
Revised for clarity

* Make a branch with your github username to work in.

> * Please mention the command and also its uses by making a dir itself in the repository :

Reviewer: ```suggestion
* Please mention the command and its uses by creating a directory within the repository itself:
```
Rephrased for clarity

>     * for instance, if "ls" is the command you are committing then name the directory file "ls" and write the command's function in a .md file inside the directory. Add a screenshot of the command and its output if needed. 

Reviewer: ```suggestion
    * For instance, if you're committing the "ls"  command, name the directory file "ls" and write the command's function in a .md file inside the directory. Add a screenshot of the command and its output if needed. 
```

> * Other than that, the contributors are free to issue a PR with anything related to Cli in this repository.

Reviewer: ```suggestion
* Other than that, contributors are free to issue a pull request (PR) with anything related to CLI in this repository.
```
Added clarification about PRs.
